### ISSUE TITLE
EXIF metadata strings containing multiple brackets have their values altered

### ISSUE THREAD

**Issue by e-adrien** [lovell/sharp#2832]
Hi,

I found something in the 0.28.3 version of sharp.
I didn't try a previous version to see if the problem was already there.

When I process files with sharp, some metadatas are changed.
For example "TEST (A (B) - [C]" becomes "TEST (A (B) - [C] (TEST (A".
I tried several files and output with the same result.

I created a small project with a test.
https://github.com/e-adrien/sharp-metadata

```bash
npx envinfo --binaries --system
  System:
    OS: Linux 5.11 Ubuntu 21.04 (Hirsute Hippo)
    CPU: (4) x64 Intel(R) Core(TM) i7-10875H CPU @ 2.30GHz
    Memory: 774.23 MB / 7.76 GB
    Container: Yes
    Shell: 5.1.4 - /bin/bash
  Binaries:
    Node: 14.17.4 - /usr/bin/node
    Yarn: 1.22.11 - /usr/bin/yarn
    npm: 7.20.5 - /usr/bin/npm
```

Let me know if you need more information.

Adrien.

**Comment by lovell** [comment]
Please see https://github.com/libvips/libvips/issues/2388

**Comment by lovell** [comment]
v0.30.0 now available with libvips v8.12.2.
